src/components/TaskCard/index.jsx

```diff
@@ -0,0 +1,23 @@
+import PropTypes from "prop-types";
+
+import "./index.scss"
+
+const TaskCard = ({ task }) => {
+  return (
+    <div className="task-card">
+      <p className="task-card__title">{task.title}</p>
+      <p>Created At: {task.createdDate}</p>
+      <button>Delete</button>
+    </div>
+  );
+};
+
+TaskCard.propTypes = {
+  task: PropTypes.shape({
+    id: PropTypes.number.isRequired,
+    title: PropTypes.string.isRequired,
+    createdDate: PropTypes.string,
```

Comment:
I will suggest to store the `createdDate` property as a Date instead of a string directly. Because later it will be very difficult to compare two dates to find the duration of a task.